samples/dotnet/ConsoleConsumer/ConsoleConsumer.csproj

```diff
@@ -1,13 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
why the change back to 2.1?

Reply:
That's what the WebJob SDK samples are built against. Wanted to follow the same rule. Figure more people will have .NET 2.1 installed too, so it lowers the barrier for a quick start.

Reply:
That's fair.